<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>weather_forecastClass</class>
 <widget class="QMainWindow" name="weather_forecastClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天气预报查询</string>
  </property>
  <property name="windowIcon">
   <iconset resource="weather_forecast.qrc">
    <normaloff>:/weather_forecast/icon/1_1.png</normaloff>:/weather_forecast/icon/1_1.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="mainbody">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="City_search_and_query">
        <item>
         <widget class="QLineEdit" name="cityInput">
          <property name="text">
           <string/>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>输入城市名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="currentWeatherLabel">
        <property name="text">
         <string>显示当前天气</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QListWidget" name="forecastList"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="weather_forecast.qrc"/>
 </resources>
 <connections/>
</ui>
